datalab/mlalpha/commands/_mlalpha.py

```diff
@@ -162,8 +163,8 @@ def mlalpha(line, cell=None):
   package_parser = parser.subcommand('feature-slice-view','View results of a ' +
                                      'FeatureSlicingPipeline, some eval metrics grouped by ' +
                                      'specified feature column values')
-  package_parser.add_argument('--file', help='The results file from FeatureSlicingPipeline',
-                              required=True)
+  package_parser.add_argument('--file', help='The results file from FeatureSlicingPipeline')
+  package_parser.add_argument('--sql', help='')
```

Comment:
missing help str. Also, package_parser is being used for two subcommands. Not an error but bad style.